crates/re_space_view/src/data_query_blueprint.rs

```diff
@@ -684,70 +540,62 @@ mod tests {
                     "/parent/skipped/child2",
                 ],
             },
-            Scenario {
-                inclusions: vec!["parent/skipped", "parent/skipped/child2", "parent/"],
-                exclusions: vec![],
-                outputs: vec![
-                    "/",
-                    "/parent/",
-                    "/parent",
-                    "/parent/skipped/",
-                    "/parent/skipped",        // Included because an exact match
-                    "/parent/skipped/child1", // Included because an exact match
-                    "/parent/skipped/child2",
-                ],
-            },
-            Scenario {
-                inclusions: vec!["parent/skipped", "parent/skipped/child2", "parent/"],
-                exclusions: vec!["parent"],
-                outputs: vec![
-                    "/",
-                    "/parent/", // Parent leaf has been excluded
-                    "/parent/skipped/",
-                    "/parent/skipped",        // Included because an exact match
-                    "/parent/skipped/child1", // Included because an exact match
-                    "/parent/skipped/child2",
-                ],
-            },
-            Scenario {
-                inclusions: vec!["parent/"],
-                exclusions: vec!["parent/skipped/"],
-                outputs: vec!["/", "/parent"], // None of the children are hit since excluded
-            },
-            Scenario {
-                inclusions: vec!["parent/", "parent/skipped/child2"],
-                exclusions: vec!["parent/skipped/child1"],
-                outputs: vec![
-                    "/",
-                    "/parent/",
-                    "/parent",
-                    "/parent/skipped/",
-                    "/parent/skipped/child2", // No child1 since skipped.
-                ],
-            },
-            Scenario {
-                inclusions: vec!["not/found"],
-                exclusions: vec![],
-                // TODO(jleibs): Making this work requires merging the EntityTree walk with a minimal-coverage ExactMatchTree walk
-                // not crucial for now until we expose a free-form UI for entering paths.
-                // vec!["/", "not/", "not/found"]),
-                outputs: vec![],
-            },
+            // Scenario {
```

Comment:
I guess these need to be rewritten because the semantics have changed a bit regarding most-specific?